docs/explanation.rst

```diff
@@ -0,0 +1,86 @@
+and `Snyk's state of open source security report 2020 <https://snyk.io/blog/10-docker-image-security-best-practices/>`_) where the findings
+state a concerning number of containers at risk deployed in cloud infrastructures.
+
+In fact, both these studies and our own assessments (dated from December 2021) show that the most popular images in Docker Hub 
+contain known vulnerabilities, with Ubuntu being the only one without any critical or high ones.
+
+.. image:: _static/container-image-vulnerabilities.png
+  :align: center
+  :width: 60%
+  :alt: Most popular container images contain known vulnerabilities
+
+Sure, consumers could venture to fix these vulnerabilities themselves, but not only this would increase the cost and scattering of images, but it wouldn't be 
+easy to accomplish due to the lack of expertise in the subject matter. The right approach is to actually fix the 
+vulnerabilities at their source! And Canonical has already started doing this. If we compare some of the Docker Official 
+container images vs some of the ones maintained by Canonical, we can verify that the latter have no high/critical vulnerabilities 
+in them!
+
+.. image:: _static/canonical-images-vulnerabilities.png
+  :align: center
+  :width: 60%
+  :alt: vulnerabilities in Official vs Canonical-maintained OCI images
+
+So this is where the motivation for a new generation of OCI images (aka ROCKs) starts - the need for more secure container images! 
+And while this need might carry the biggest weight in the container users' demands, other values come into play when 
+selecting the best container image, such as:
+ 
+ - stability
+ - size
+ - compliance
+ - provenance
+
+You can find these values and their relevance in this report: https://juju.is/cloud-native-kubernetes-usage-report-2021#selection-criteria-for-container-images.
+
+This brings us to the problem statement behind ROCKs::
+
+    How might we redesign secure container images for Kubernetes developers and application maintainers, considering the Top 10 Docker images are full of vulnerabilities, except Ubuntu?
+
+A ROCK is:
+
+ - **secure** and **stable**: based on the latest and greatest Ubuntu releases;
+ - **OCI-compliant**: compatible with all the popular container management tools (Docker, Kubernetes, etc.);
+ - **dependable**: built on top of Ubuntu, with a predictable release cadence and timely security updates;
+ - **production-grade**: tested and secured by default.
+
+
+Do I need to use Rockcraft?
+===========================
+
+If you want to build a proper ROCK, yes, we'd recommend you do. This is not to say you wouldn't be able to build ROCK-like container images 
+with your own tools, but Rockcraft has been developed precisely to offer an easy way to build production-grade container images.
+
+Furthermore, Rockcraft is built on top of existing concepts and within the same family as Snapcraft and Charmcraft, 
```

Comment:
Maybe we should link to the Snapcraft docs here? They are the most comprehensive ones I think. Something like https://snapcraft.io/docs/snapcraft-overview